src/app/components/properties/properties.component.spec.ts

```diff
@@ -33,14 +32,14 @@ describe('PropertiesComponent', () => {
   beforeAll(() => resetSpies([propertiesServiceSpy]));
   afterEach(() => resetSpies([propertiesServiceSpy]));
 
-  beforeEach(() => {
+  configureTestSuite(() => {
     TestBed.configureTestingModule({
-      declarations: [PropertiesComponent, CardComponent],
+      declarations: [],
       imports: [
         AppModule,
         HttpClientModule,
         NgxSpinnerModule,
-        AuthLayoutModule,
+        // AuthenticationModule,
```

Comment:
@kdaglas there's another comment that seems to have evaded your attention.

Reply:
on it... thanks